Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -473,6 +473,11 @@ void HexEditor::keydown_event(GUI::KeyEvent& event)
         return;
     }
 
+    if (event.key() == KeyCode::Key_Home) {
```

Comment:
No need to include "As the title says - "
And cursors are not gendered in English :)
s/him/it/